scripts/index.js

```diff
@@ -1,147 +0,0 @@
-class CustomerInventory extends Wines {
-    constructor() {
-        super();
-    }
-
-    // the customer wine only contains wine name and quantity
-    addWine(name, quantity) {
-        // add a number of wine into the database: check the quantities against the inventory
-        var ret = RuturnCode.ERROR;
-        var quantities = gInventory.get(name, quantity);
-        if (typeof quantities !== 'undefined' && quantity < quantities) {
-            var customerQuantities = this.wines.get(name);
-            if (typeof customerQuantities !== 'undefined') {
-                this.wines.set(name, quantities + quantity);
-            }
-        }
-        return ret;
-    }
-}
-
-class Inventory extends Wines {
-    constructor() {
-        super();
-        this.wines.set(
-            'Russia Wine 1',
-            new Wine(
-                'Russia Wine 1',
-                '/images/russia/1.png',
-                22.22,
-                'Drunk Russia Man',
-                100
-            )
-        );
-        this.wines.set(
-            'China Wine 1',
-            new Wine(
-                'China Wine 1',
-                '/images/china/1.png',
-                22.22,
-                'Drunk China Man',
-                100
-            )
-        );
-    }
-
-    getQuantities(name) {
-        var quantities = 0;
-        quantiies = this.wines.get(name);
-        return quantities;
-    }
-}
-
-class Customer {
-    constructor(name, birthday, email, phone, address, password) {
-        this.name = name;
-        this.birthday = birthday;
-        this.email = email;
-        this.phone = phone;
-        this.address = password;
-        this.wines = new Map(); // wine list but the quantity is the purchased wine quantities
-    }
-
-    addWine(name, quantity) {
-        return addWine(name, quantity);
-    }
-
-    removeWine(wineName, quantity) {}
-}
-
-class Customers {
-    constructor() {
-        this.customers = new Map();
-    }
-
-    add(customer) {
-        this.customers.set(customer.name, customer);
-    }
-
-    saveJSONFile() {
-        // convert Map to JSON string
-        // const jsonStr = JSON.stringify(Array.from(gCustomers.customers.entries()));
-        // // save JSON to file
-        // const fs = require("fs");
-        // fs.writeFile(CustomersFile, jsonStr, err => { console.log("Error happens while saving customers.json") })
-    }
-
-    readJSONFile() {
-        // // Read JSON file
-        // const fs = require("fs");
-        // fs.readFile(CustomersFile, (err, data) => {
-        //     if (err) throw err;
-        //     // convert JSON string to Map
-        //     var obj = JSON.parse(jsonStr, (key, value) => {
-        //         return value;
-        //     });
-        // });
-    }
-}
-
-let gCustomers = new Customers();
-let gInventory = new Inventory();
-
-function test() {
-    gCustomers.add(new Customer('Karen Mok', '1900', '', '', '', ''));
-    gCustomers.saveJSONFile();
-}
-
-test();
```

Comment:
I notice our index.js file is empty, may want to work together to see want code can be moved here once we all get together.